src/commands/twitter.js

```diff
@@ -2,6 +2,7 @@ import debug from 'debug';
 import * as TwitterClient from 'twitter-node-client';
 import config from './../../config';
 import {Command} from './command.js';
+import he from 'he';
```

Comment:
Nit - can we bump this up? I like to include external packages above internal ones